.github/workflows/ci.yml

```
@@ -238,102 +266,103 @@ jobs:
       shell: bash
 
   windows-mingw:
-    name: 'Windows (mingw-w64, Debug, 11, x86_64, windows-2019)'
+    name: 'Windows (mingw-w64, x86_64, windows-2019)'
     runs-on: windows-2019
     defaults:
       run:
         shell: msys2 {0}
+    strategy:
+      matrix:
+        build_type: ['Debug', 'Release']
+
     steps:
     - name: 'Check Out'
       uses: actions/checkout@v3
 
     - name: 'Setup'
       uses: msys2/setup-msys2@v2
       with:
-        install: mingw-w64-x86_64-gcc mingw-w64-x86_64-cmake make
+        install: mingw-w64-x86_64-gcc mingw-w64-x86_64-cmake make ccache
```

This should fix the `windows-mingw` job, it doesn't seem to mind which direction the slashes go:
```suggestion
        install: mingw-w64-x86_64-gcc mingw-w64-x86_64-cmake make mingw-w64-x86_64-ccache
```
see [mingw-w64-x86_64-ccache](https://packages.msys2.org/package/mingw-w64-x86_64-ccache) vs [ccache](https://packages.msys2.org/package/ccache).

Also L298 might not be needed.